pkg/eraser/helpers.go

```diff
@@ -6,7 +6,11 @@ import (
 	util "github.com/Azure/eraser/pkg/utils"
 )
 
+var removed int
```

Comment:
I don't think this needs to be a global. You can just provide it as a return value from `removeImages`.